features/shopcarts.feature

```diff
@@ -75,4 +75,12 @@ Scenario: Add an item to a shopcart, and then delete the entire shopcart
     Then I should see the message "Success! Added 5 iPod MAX to this shopcart"
     When I paste the "Shopcart_ID_top" field
     And I press the "Delete" button
-    Then I should see the message "Shopcart deleted"
\ No newline at end of file
+    Then I should see the message "Shopcart deleted"
+
+Scenario: List all items in a shopcart
+    When I visit the "Home Page"
+    And I set the "Shopcart_ID_bottom" to "1"
```

Comment:
You need to actually add a shopcart and items here. Look to lines 60-74 to copy and paste and edit the field names as you see fit.

Reply:
I saw the issue, and it only mentions that we are testing showing the items in the shopcart. I think there is another story for adding an item to a shopcart.

Reply:
Yes I already did that story. You just need to copy the code so you can test your story. If you run behave on your story it fails. More details in the slack.

Reply:
Now is ready to merge! thank you Ross!!